seatunnel-connectors-v2/connector-hive/src/main/java/org/apache/seatunnel/connectors/seatunnel/hive/sink/HiveSink.java

```diff
@@ -207,4 +71,21 @@ public void prepare(Config pluginConfig) throws PrepareFailException {
         return Optional.of(
                 new HiveSinkAggregatedCommitter(pluginConfig, dbName, tableName, hadoopConf));
     }
+
+    @Override
+    public Optional<SaveModeHandler> getSaveModeHandler() {
+        if (pluginConfig.hasPath(SCHEMA_SAVE_MODE.key())
+                || pluginConfig.hasPath(DATA_SAVE_MODE.key())) {
+            ReadonlyConfig readonlyConfig = ReadonlyConfig.fromConfig(pluginConfig);
+            SchemaSaveMode schemaSaveMode = readonlyConfig.get(SCHEMA_SAVE_MODE);
+            DataSaveMode dataSaveMode = readonlyConfig.get(DATA_SAVE_MODE);
+            String custom_sql = readonlyConfig.get(CUSTOM_SQL);
+            new ReadonlyConfigParser().parse(readonlyConfig);
+            HiveCatalog catalog = new HiveCatalog(pluginConfig);
+            return Optional.of(
+                    new DefaultSaveModeHandler(
```

Comment:
When people use Hive to automatically create tables, they may need to specify location, field delimiters, storage formats, etc. So I think Hive's automatic table creation should mimic Doris by specifying a table creation template for creation, and by modifying the template to specify table properties when needed.